site/cli2md/cli2md.go

```diff
@@ -236,13 +236,27 @@ func generateSubcommands(w io.Writer, dir, sectionTitle string, cmds []*kingpin.
 		}
 
 		if first {
-			fmt.Fprintf(w, "\n### %v\n\n", sectionTitle)
+			sectionTitleCutoffIndex := len(sectionTitle)
+			if sectionTitle[sectionTitleCutoffIndex-1] == '.' {
+				sectionTitleCutoffIndex -= 1
+			}
+
+			fmt.Fprintf(w, "\n### %v\n\n", sectionTitle[:sectionTitleCutoffIndex])
 
 			first = false
 		}
 
+		helpSummaryCutoffIndex := strings.Index(c.Help, "\n")
```

Comment:
strings.TrimSuffix() again?

Reply:
sorry, my bad this is something more like `strings.Split(..., "\n")[0]`

Reply:
I chose to not use Split because I thought this has slight performance benefit due to only needing the first token. But I admit I didn't do any benchmark to prove. So if you prefer concise code, I'll change.

Reply:
Never mind. Found `strings.SplitN`